go.mod

```diff
@@ -2,19 +2,27 @@ module github.com/celestiaorg/celestia-node
 
 go 1.16
 
+replace github.com/ipfs/go-verifcid => github.com/celestiaorg/go-verifcid v0.0.1-lazypatch
+
 require (
 	github.com/BurntSushi/toml v0.4.1
 	github.com/celestiaorg/celestia-core v0.0.2-0.20210924001615-488ac31b4b3c
+	github.com/celestiaorg/nmt v0.7.0
 	github.com/celestiaorg/rsmt2d v0.3.0
-	github.com/ipfs/go-bitswap v0.4.0
+	github.com/ipfs/go-bitswap v0.3.4
+	github.com/ipfs/go-block-format v0.0.3
 	github.com/ipfs/go-blockservice v0.1.7
+	github.com/ipfs/go-cid v0.0.7
 	github.com/ipfs/go-datastore v0.4.6
 	github.com/ipfs/go-ds-badger2 v0.1.1
 	github.com/ipfs/go-ipfs-blockstore v0.1.6
 	github.com/ipfs/go-ipfs-exchange-interface v0.0.1
+	github.com/ipfs/go-ipld-cbor v0.0.5 // indirect
 	github.com/ipfs/go-ipld-format v0.2.0
 	github.com/ipfs/go-log/v2 v2.3.0
-	github.com/ipfs/go-merkledag v0.4.0
+	github.com/ipfs/go-merkledag v0.3.2
+	github.com/ipfs/go-peertaskqueue v0.4.0 // indirect
+	github.com/ipld/go-ipld-prime v0.11.0 // indirect
```

Comment:
Well, we should double-check why this dependency is here. That's basically an import of the newer IPLD stack that we are not using.